landoapi/commit_message.py

```diff
@@ -204,4 +204,4 @@ def bug_list_to_commit_string(bug_ids: List[str]) -> str:
     if not bug_ids:
         return "No bug"
 
-    return f"Bug {', '.join(bug_ids)}"
+    return f"Bug {', '.join(dict.fromkeys(bug_ids))}"
```

Comment:
```suggestion
    return f"Bug {', '.join(sorted(set(bug_ids)))}"
```

Hey @mathewhodson, I'd like to see this land next week. Can you make the above change and update this PR?